AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/OrganizationController.cs

```diff
@@ -76,6 +62,16 @@ public IActionResult Edit(int id)
         [ValidateAntiForgeryToken]
         public IActionResult Edit(OrganizationEditModel organization)
         {
+            if (organization == null)
+            {
+                return HttpBadRequest();
+            }
+
+            if (!User.IsUserType(UserType.SiteAdmin))
+            {
+                return HttpUnauthorized();
+            }
```

Comment:
I think this check is redundant to the `[Authorize("SiteAdmin")]` at the top of the class.